packages/teams-js/src/public/authentication.ts

```diff
@@ -281,7 +282,7 @@ export namespace authentication {
     height = Math.min(height, Communication.currentWindow.outerHeight - 200);
     // Convert any relative URLs into absolute URLs before sending them over to the parent window
     const link = document.createElement('a');
-    link.href = authenticateParameters.url;
+    link.href = authenticateParameters.url.replace('{oauthRedirectMethod}', 'web');
```

Comment:
>link.href = authenticateParameters.url.replace('{oauthRedirectMethod}', 'web'); [](http://example.com/codeflow?start=4&length=79)

Not something that should hold up the merge, but @AE-MS have you seen something like this before? Came from this change: [https://github.com/OfficeDev/microsoft-teams-library-js/pull/750](https://github.com/OfficeDev/microsoft-teams-library-js/pull/750)

Reply:
@MarcinJuraszek want to look at this?